src/modules/portfolio/components/transaction-linked/transaction-linked.jsx

```diff
@@ -40,7 +40,7 @@ export default class LinkedTransaction extends Component {
           ref={(metaWrapper) => { this.metaWrapper = metaWrapper }}
           className={ToggleHeightStyles['toggle-height-target']}
         >
-          <TransactionMeta meta={p.transaction.meta} />
+          <TransactionMeta meta={p.transaction.meta} networkId={p.networkId} />
```

Comment:
Re: passing networkId — it’s totally valid in a situation like this to have a container component _just_ for this component to consume `networkId` so that all depending components don’t have to pass it.

Reply:
That’s actually the direction we’re moving — where necessary, we’re doing 1-to-1 container/component pairings to reduce unnecessarily deeply passed props + improve reusability of the components.

Reply:
Not sure how to do a 1 to 1 with passing networkId to only TrasactionMeta and not pass it through the other components. Can you explain or send me to a link.

Reply:
Sure!

Let’s plan to setup a call sometime today + I can walk you through it.